docs/user-guide/slicing.ipynb

```diff
@@ -409,9 +409,9 @@
    "outputs": [],
    "source": [
     "da = sc.DataArray(\n",
-    "    data = sc.Variable(dims=['x'], values=np.random.random(7)),\n",
+    "    data = sc.array(dims=['x'], values=np.random.random(7)),\n",
     "    coords={\n",
-    "        'x': sc.Variable(['x'], values=np.linspace(1.0, 2.0, num=8), unit=sc.units.m)})\n",
+    "        'x': sc.array(dims=['x'], values=np.linspace(1.0, 2.0, num=8), unit=sc.units.m)})\n",
     "da"
```

Comment:
Are you planning on making this the preferred approach in the documentation. In which case are you going to do that for all docs here or as a follow up? We should take a consistent approach in the docs.

Reply:
I think the plan currently is to implement it and see what we think in practice. I have updated the single notebook so that people can have a look at what it would look like, this may well not end up being merged though. 

Whether and to what extent we continue with this approach still needs to be decided.